tests/disas/gc/drc/array-len.wat

```diff
@@ -1,5 +1,5 @@
 ;;! target = "x86_64"
-;;! flags = "-W function-references,gc"
+;;! flags = "-W function-references,gc -C collector=drc"
```

Comment:
To confirm this technically isn't necessary right? (to specify `-C collector=drc`)

I think it's good to have in the test for future-proofing, but I want to double-check that as-is in isolation this isn't needed if we make no more changes to wasmtime in the future

Reply:
Correct, this is just future-proofing for if/when we change the default collector.